plugin/src/helpers/functions.ts

```diff
@@ -73,6 +75,7 @@ export const setupImageFunction = async ({
   await writeJSON(join(functionDirectory, 'imageconfig.json'), {
     ...imageconfig,
     basePath: [basePath, IMAGE_FUNCTION_NAME].join('/'),
+    remotePatterns,
```

Comment:
I'll add a cypress test in this PR, however because `remotePatterns` is only available in canary (and Next.js won't add it to the config we use to build `imageconfig.json`) it'll fail on `latest`. I've added a todo in https://github.com/netlify/pod-ecosystem-frameworks/issues/112 to add a test for `imageconfig.json` in canary. :)